app/views/Coach/Clients/index.js

```diff
@@ -121,28 +131,32 @@ class Clients extends Component {
               </GridList>
             </div>
             </Tab>
-            <Tab label="Zablokowani" >
+            <Tab label="Zablokowani" style={styles.tabtab} >
             <div style={styles.root}>
               <GridList
-                cellHeight={350}
+                cellHeight={250}
                 style={styles.gridList}
+                cols={3}
               >
                 <Subheader>Zablokowani klienci</Subheader>
                 {this.state.blockedClientsList.map((tile) => (
-                  <GridTile
-                    key={tile.id}
-                    title={<span>{tile.name} {tile.last_name}</span>}
-                    //todo - data zablokowania
-                    subtitle={<span>zablokowany: <b>{tile.updated_at}</b></span>}
-                    actionIcon={<IconButton><InfoIcon /></IconButton>}
-                  >
-                    <img src='../../../assets/images/avatar.png' />
-                  </GridTile>
+                  <Link to={'/coach/clients/'+tile.id} >
+                    <GridTile
+                      key={tile.id}
+                      title={<span>{tile.name} {tile.last_name}</span>}
+                      //todo - data zablokowania
```

Comment:
Czemu? Co to znaczy data zablokowana? Jest jakiś problem z backendem?

Reply:
ask Diana

Reply:
Nie znalazłam info o dacie zablokowania, będzie tu wstawiona data ostatniej aktualizacji klienta

Reply:
Teraz widzę że jest wstawione updated_at :D więc do usunięcia sam komentarz